public/wp-content/plugins/ccs-custom/library/options-page.php

```diff
@@ -195,3 +195,14 @@
     'updated_message'	=> __("Homepage Components Updated", 'acf'),
 
 ) );
+
+$option_page = acf_add_options_page(array(
+    'page_title'    => __('Redirection'),
```

Comment:
Just a small observation. Curios why to do it like this : `__('Redirection')` instead of `'Redirection'`? What the difference exactly?

Reply:
I am not too sure, I was just following the doc from ACF: https://www.advancedcustomfields.com/resources/acf_add_options_page/